### PR TITLE
1d serial

### PR DESCRIPTION
**Pull request by Pranavug**
1d sequential implementation

### CODE REVIEW

File: 1d_serial/print_time_results.hpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
+//  Copyright (c) 2014 Hartmut Kaiser
```
Comment (diehlpk) [inline 1d_serial/print_time_results.hpp:1]: Please add your name to the copyright

File: 1d_serial/1d_nonlocal_serial.cpp [hunk 2]
```diff
@@ -0,0 +1,174 @@
+#include <hpx/hpx_init.hpp>
```
Comment (diehlpk) [inline 1d_serial/1d_nonlocal_serial.cpp:1]: Please add the copyright header with your name to this file. If you have copied part of this file from the HPX repo, please add the copyright header from there.

File: 1d_serial/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.3.2)
```
Comment (diehlpk) [inline 1d_serial/CMakeLists.txt:1]: Please add the copyright header with your name

**Comment by Pranavug** [comment]
Hello Patrick, I have made the required changes. 
Please let me know if anything else is to be changed

**Pull request merged**